core/src/autogluon/core/utils/try_import.py

```diff
@@ -53,7 +53,7 @@ def try_import_mxnet():
 
 
 def try_import_ray() -> ModuleType:
-    RAY_MAX_VERSION = '2.3.0'
+    RAY_MAX_VERSION = '3.0.0'
```

Comment:
add inline comment that this is a non-inclusive max version

Reply:
This is not meant to be here.. It's needed to enable me easily test the distributed functionality. Will revert